e2e-tests/playwright/tests/functional/channels/account_settings/profile/popover_fields.spec.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, test} from '@e2e-support/test_fixture';
+import {createRandomUser} from '@e2e-support/server';
+
+test('MM-T53377 Profile popover should show correct fields after at-mention autocomplete', async ({pw, pages}) => {
+    // # Initialize with specific config and get admin client
+    const {user, adminClient, team} = await pw.initSetup();
+
+    await adminClient.patchConfig({
+        PrivacySettings: {
+            ShowEmailAddress: false,
+            ShowFullName: false,
+        },
+    });
+
+    // # Create and add another user using admin client
+    const testUser2 = await adminClient.createUser(createRandomUser(), '', '');
+    await adminClient.addToTeam(team.id, testUser2.id);
+
+    // # Log in as user in new browser context
+    const {page} = await pw.testBrowser.login(user);
+
+    // # Visit default channel page
+    const channelPage = new pages.ChannelsPage(page);
+    await channelPage.goto();
+    await channelPage.toBeVisible();
+
+    // # Send mentions quickly
+    await channelPage.centerView.postCreate.postMessage(`@${user.username} @${testUser2.username}`);
+
+    // # Open profile popover for current user
+    const firstMention = channelPage.centerView.container.getByText(`@${user.username}`, {exact: true});
+    await firstMention.click();
+
+    // * Verify all fields are visible for current user
+    const popover = channelPage.userProfilePopover;
+    await expect(popover.container.getByText(`@${user.username}`)).toBeVisible();
```

Comment:
Test failed here and I can see it failed in local too (flaky). One option is to add visibility check first that the container (though will not work for `#user-profile-popover-portal`) or one of its field (ex. username with `#userPopoverUsername`).

Reply:
Actually, it wasn't a flaky test, it was a merge problem, something was changed 3 days ago, that is the reason why it was failing. now it is fixed.